docs/developer/architecture/logs.rst

```diff
@@ -0,0 +1,120 @@
+Log Centralisation
+==================
+
+Context
+-------
+
+MetalK8s value is to provide, out of the box, some services in order to ease
+the monitoring and operation of the platform as well as workload running on top
+of it.
+
+MetalK8s provides monitoring service with Prometheus and AlertManager in
+order to expose MetalK8s and workloads metrics and alerts.
```

Comment:
Could read better this way?

`Metalk8s provides a monitoring stack consisting of Prometheus and Alertmanager
with a goal to expose and monitor metrics and/or alerts for both
control-plane and workload-plane components.`

Reply:
updated